### PR TITLE
Adds an option ENABLE_UNICODE to winbuild/Makefile.vc

### PR DESCRIPTION
**Pull request by apique13**

### CODE REVIEW

File: winbuild/MakefileBuild.vc [hunk 1]
```diff
@@ -485,14 +485,18 @@ LFLAGS = $(LFLAGS) $(LFLAGS_PDB)
 CFLAGS = $(CFLAGS) /DCURL_WITH_MULTI_SSL
 !ENDIF
 
+!IF "$(USE_UNICODE)"=="true"
+CFLAGS = $(CFLAGS) /DUNICODE
```
[the remark this answers is not in the document]
Comment (apique13) [reply winbuild/MakefileBuild.vc]: Yes maybe. I saw in curl_setup.h that if one was declared the other will be defined too : 
#  if defined(UNICODE) && !defined(_UNICODE)
#    define _UNICODE
#  endif
#  if defined(_UNICODE) && !defined(UNICODE)
#    define UNICODE
#  endif

[the remark this answers is not in the document]
Comment (MarcelRaad) [reply winbuild/MakefileBuild.vc]: That's only a workaround for wrong usage. Please add `_UNICODE` too. Apart from that, I think this is good to merge.

**Comment by bagder** [comment]
The red CI builds seem unrelated (and fixed in master already).

**Comment by MarcelRaad** [comment]
Thanks! Squashed and pushed with a slightly edited commit message.

**Pull request closed without merging**